ext/fluxnet_simulations/forcing.jl

```diff
@@ -23,7 +23,9 @@ precipitation that is in snow (if split_precip ==true), and (5)
 making the TimeVaryingInput objects.
 
 Note that the TimeVaryingInput objects can be used to interpolate in time,
-which is why we drop missing data.
+which is why we drop missing data. For the timestamp of the observation, we use
```

Comment:
for the forcing, using the midpoint makes the most sense, since the average is over the window